examples/create-collection.py

```diff
@@ -2,7 +2,13 @@
 
 from pestuary.collections import collection_create
 
-name = "Test"
-description = "Helpful description here"
-response = collection_create(name, description)
-print(response)
+
+def example_collection_create():
```

Comment:
Same as above. Putting this code in a function does not make it more readable and it confuses the example